### PR TITLE
vdr-addon: update addon

### PR DESCRIPTION
**Pull request by CvH**
bump for https://github.com/LibreELEC/LibreELEC.tv/pull/1475

### CODE REVIEW

**Comment by CvH** [comment]
good to merge

File: packages/addons/service/vdr-addon/package.mk [hunk 1]
```diff
@@ -18,11 +18,11 @@
 ################################################################################
 
 PKG_NAME="vdr-addon"
-PKG_VERSION="8.0"
+PKG_VERSION="2.2"
```
Comment (MilhouseVH) [inline packages/addons/service/vdr-addon/package.mk]: Is this likely to cause a problem for updates?

Comment (CvH) [inline packages/addons/service/vdr-addon/package.mk]: this version is more or less just for the build dir that it isn't `vdr-addon-` for nothing else it is used

Comment (MilhouseVH) [inline packages/addons/service/vdr-addon/package.mk]: OK thanks

**Pull request merged**